exchanges/localbitcoins/localbitcoins_wrapper.go

```diff
@@ -248,18 +248,20 @@ func (l *LocalBitcoins) UpdateOrderbook(ctx context.Context, p currency.Pair, as
 		return book, err
 	}
 
+	book.Bids = make(orderbook.Items, len(orderbookNew.Bids))
 	for x := range orderbookNew.Bids {
-		book.Bids = append(book.Bids, orderbook.Item{
+		book.Bids[x] = orderbook.Item{
```

Comment:
v small chance of application death if localbitcoins ever sends a zero value price